lib/LedgerSMB/Request/Error.pm

```diff
@@ -32,11 +32,11 @@ String to send as error message
 
 =cut
 
-has msg => (is => 'ro', isa => 'Str', required => 1);
+has msg => (is => 'ro', isa => 'Object', required => 1);
```

Comment:
Because we do not always pass a Str.

Le 2016-04-22 15:00, Erik Huelsmann a écrit :
>
> In lib/LedgerSMB/Request/Error.pm
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60787421>:
>
> > @@ -32,11 +32,11 @@ String to send as error message
> >  
> >  =cut
> >  
> > -has msg => (is => 'ro', isa => 'Str', required => 1);
> > +has msg => (is => 'ro', isa => 'Object', required => 1);
>
> Why did you have to change this to an Object? I can't see why $msg
> should be an object in the code above: if it's an instance of
> LedgerSMB::Request::Error, the constructor isn't called for another
> wrapping.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60787421>
>